src/ui/components/DevTools.tsx

```diff
@@ -60,6 +64,11 @@ function _DevTools({
       setExpectedError(expectedError);
     }
   }, [recordingId, expectedError]);
+  useEffect(() => {
+    if (!recordingQueryLoading && !userInfo.loading && recording) {
+      LogRocket.createSession(recording, userInfo, auth);
```

Comment:
We only need to create a session in DevTools, so I moved this code here.

Reply:
Is that because we don't want LogRocket sessions for the library view?

Reply:
Yeah. I think we do want to track other flows like the library eventually, but the current implementation keeps that from happening anyway. So logrocket behaves the same before and after this PR.